### PR TITLE
Add a new CODEOWNERS file, and add a few owners

### PR DESCRIPTION
**Pull request by dnephin**
Starting with just a few patterns, let me know if you'd like me to add more to this PR.

cc @albers @vdemeester to make sure you're ok with this.

Being listed in this file tells github to notify you when anyone opens a PR which touches files which match the pattern.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Github code owners
+# See https://github.com/blog/2392-introducing-code-owners
+
+cli/compose/*              @dnephin @vdemeester
+contrib/completion/bash/*  @albers
```
Comment (thaJeztah) [inline .github/CODEOWNERS]: can add @sdurrheimer for contrib/completion/zsh

Comment (thaJeztah) [inline .github/CODEOWNERS]: perhaps `docs/` -> @mstanleyjones, @vdemeester  and me

Comment (dnephin) [inline .github/CODEOWNERS]: updated

**Comment by vieux** [comment]
LGTM ping @albers

**Comment by albers** [comment]
LGTM

**Pull request merged**